dev/freestanding/cosri/docker-compose.yaml

```diff
@@ -8,12 +8,12 @@ services:
       REACT_APP_CONF_API_URL: "https://backend.${BASE_DOMAIN:-localtest.me}"
       REACT_APP_DASHBOARD_URL: "https://dashboard.${FEMR_BASE_DOMAIN:-localtest.me}"
 
-      REACT_APP_DISABLE_SECTION_HISTORICALTREATMENTS: "true"
-      REACT_APP_DISABLE_SECTION_PERTINENTMEDICALHISTORY: "true"
-      REACT_APP_DISABLE_SECTION_NONPHARMACOLOGICTREATMENTS: "true"
-      REACT_APP_DISABLE_SECTION_RISKCONSIDERATIONS: "true"
-      REACT_APP_DISABLE_SECTION_URINEDRUGSCREENS: "true"
-      REACT_APP_DISABLE_SECTION_NALOXONEMEDICATIONS: "true"
+      REACT_APP_SECTION_HISTORICALTREATMENTS: hidden
+      REACT_APP_SECTION_PERTINENTMEDICALHISTORY: hidden
+      REACT_APP_SECTION_NONPHARMACOLOGICTREATMENTS: hidden
+      REACT_APP_SECTION_RISKCONSIDERATIONS: hidden
```

Comment:
@ivan-c Ivan, sorry I didn't catch this but I think we want to show Risk Consideration section as it has sections we want to show - so I think we should remove `REACT_APP_DISABLE_SECTION_RISKCONSIDERATIONS` .  But I think we want to hide the sub-sections e.g. NALOXONE Medication and URINE DRUG SCREEN within that section

Reply:
@achen2401 
Done! Can you re-review when you get a chance?

Reply:
@ivan-c thank you.  I **think**  Lydia wants the **Pertinent Medical History** section shown still, as it contains an associated flag for patient over age of 65.  I'd remove `REACT_APP_SECTION_PERTINENTMEDICALHISTORY` for now.

Reply:
@achen2401
Done! Can you re-review when you get a chance?